src/client/pages/OfferNew/Introduction/Sidebar/CancellationOptions.tsx

```diff
@@ -1,47 +1,39 @@
 import styled from '@emotion/styled'
-import { colorsV3 } from '@hedviginsurance/brand'
 import { format } from 'date-fns'
```

Comment:
nit: make more explicit

```suggestion
import { format as formatDate } from 'date-fns'
```